src/util/ConfigManager/ConfigManager.h

```diff
@@ -269,8 +275,15 @@ class ConfigManager {
     move constructor. Which is why, we can't just return the `ConfigOption`
     we created here.
     */
-    return configurationOptions_.at(createJsonPointerString(pathToOption))
-        .get();
+    return *configurationOptions_.at(createJsonPointerString(pathToOption))
+                .get();
   }
+
+  /*
+  @brief Provide a range of tuples, that hold references to the key value pairs
+  in `configurationOptions_`, but with the pointer dereferenced.
+  */
+  auto getDereferencedConfigurationOptionsView();
+  auto getDereferencedConfigurationOptionsView() const;
```

Comment:
I think you can simply call those functions `configurationOptions()`, 
As they return "normal" access to the hash map without the implementation detail of the `unique_ptr`s

Reply:
Done, see [3559c40](https://github.com/ad-freiburg/qlever/pull/1040/commits/3559c406975bdcba59d64ce1af45a6187e3e431b).